resources/views/settings/subscription/index.blade.php

```diff
@@ -58,9 +69,15 @@
             <span class="slider subscription-period-slider"></span>
         </label>
         <span>{{ __('tiers.toggle.yearly') }}</span>
+
+        <select class="custom-select custom-select-lg mb-3" id="currency-selector">
+            <option selected>Currency</option>
+            <option value="eur">EUR</option>
+            <option value="usd">USD</option>
```

Comment:
The USD option in the dropdown isn't selected by default when outside of the few countries using EUR